lib/jxl/common.h

```diff
@@ -25,6 +25,12 @@
 namespace jxl {
 // Some enums and typedefs used by more than one header file.
 
+#ifdef MEMORY_SANITIZER
+// Chosen so that kSanitizerSentinel is four copies of kSanitizerSentinelByte.
+constexpr uint8_t kSanitizerSentinelByte = 0x48;
+constexpr float kSanitizerSentinel = 205089.125f;
```

Comment:
what do you think about using a NaN instead?

Reply:
I thought about it, but I am not sure it is a good idea - doing SIMD operations on NaNs (and subnormals) can have weird performance problems, and it is possible that sooner or later we'll end up using the sentinel in places where we shouldn't.

Also, sanitizers are likely to complain more about NaN values in padding, and possibly create false positives...